src/Control/Effect/Pure.hs

```diff
@@ -10,7 +10,6 @@ module Control.Effect.Pure
 import Control.Applicative
 import Control.Monad.Fix
 import Data.Coerce
-import Data.Function (fix)
```

Comment:
Another instance of a redundant import that ghc 8.8.1 is smarter about catching—`Control.Monad.Fix` exports the same function.